db/schema.rb

```diff
@@ -46,10 +46,11 @@
     t.string "action_link_url"
     t.string "action_link_content"
     t.string "action_link_summary"
+    t.string "sub_heading"
```

Comment:
I don't think this should be here. I think you need to rollback the last migration first.

Reply:
Jinx 😀